bridge/src/bridge/withdraw_relay.rs

```diff
@@ -46,7 +54,15 @@ fn signatures_payload(foreign: &foreign::ForeignBridge, signatures: u32, my_addr
 	}))
 }
 
-fn withdraw_relay_payload(home: &home::HomeBridge, signatures: Vec<Bytes>, message: Bytes) -> Bytes {
+fn message_value_sufficient_payload(home: &home::HomeBridge, message: &Bytes) -> Bytes {
+	assert_eq!(message.0.len(), 84, "ForeignBridge never accepts messages with len != 84 bytes; qed");
+	home
+		.functions()
+		.is_message_value_sufficient_to_cover_relay()
+		.input(message.0.clone()).into()
+}
+
+fn withdraw_relay_payload(home: &home::HomeBridge, signatures: &Vec<Bytes>, message: &Bytes) -> Bytes {
```

Comment:
it's better to `&[Bytes]` instead of `&Vec<Bytes>`

Reply:
indeed. i'll change it